utils/db/setup.ts

```diff
@@ -0,0 +1,18 @@
+import conn from '.';
+
+export const getGithubReposFromDbForUserId = async (user_id: string, org: string, provider: string): Promise<string[]> => {
```

Comment:
```suggestion
export const getGithubReposFromDbForUserId = async (userId: string, org: string, provider: string): Promise<string[]> => {
```

when defining your own function, since you don't have any dependency on any other thing, you should always use camelCase parameter names.

Reply:
Got it